src/screens/FeedScreen/Header.js

```diff
@@ -216,18 +290,28 @@ const _renderProfileNormal = ({
             onPress={navigateToProfile}
             style={[styles.containerFeedProfile, {paddingBottom: 5}]}>
             <View style={[styles.containerFeedName, {alignItems: 'flex-end'}]}>
-              <Text style={styles.feedUsername}>
-                {username || StringConstant.feedDeletedUserName}
-              </Text>
+              <View style={styles.contentFeedName}>
+                <Text style={styles.feedUsername}>
+                  {username || StringConstant.feedDeletedUserName}
+                </Text>
+                {!isSelf && (
+                  <React.Fragment>
+                    <View style={styles.point} />
+                    <TouchableOpacity onPress={() => onPressFollUnFoll(isFollow)}>
+                      <Text style={isFollow ? styles.textFollowing : styles.textFollow}>
+                        {isFollow ? 'Following' : 'Follow'}
+                      </Text>
+                    </TouchableOpacity>
+                  </React.Fragment>
+                )}
+              </View>
 
               <GlobalButton
                 buttonStyle={{marginLeft: 'auto', paddingBottom: 0, alignSelf: 'center'}}
-                onPress={onHeaderOptionClicked}>
-                {hideThreeDot ? null : (
-                  <View style={{zIndex: 1000}}>
-                    <ElipsisIcon width={4} height={14} fill={colors.blackgrey} />
-                  </View>
-                )}
+                onPress={() => refSheet.current.open()}>
+                <View style={{zIndex: 1000}}>
+                  <ElipsisIcon width={4} height={14} fill={colors.blackgrey} />
+                </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The user interface has been updated to include a follow/unfollow button next to the username. This is a significant change and should be tested thoroughly to ensure it works as expected across different screen sizes and devices.
```diff
+               <View style={styles.contentFeedName}>
+                 <Text style={styles.feedUsername}>
+                   {username || StringConstant.feedDeletedUserName}
+                 </Text>
+                 {!isSelf && (
+                   <React.Fragment>
+                     <View style={styles.point} />
+                     <TouchableOpacity onPress={() => onPressFollUnFoll(isFollow)}>
+                       <Text style={isFollow ? styles.textFollowing : styles.textFollow}>
+                         {isFollow ? 'Following' : 'Follow'}
+                       </Text>
+                     </TouchableOpacity>
+                   </React.Fragment>
+                 )}
+               </View>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->